make_release.rb

```diff
@@ -372,16 +404,20 @@ def zip_package(target, target_name, target_folder, target_file)
 
 def package(target, target_name, target_folder, target_file)
   if target_mac? target
-    puts 'Including licenses in mac .zip'
+    if @options[:special]
+      SPECIAL_BUILDS[@options[:special]][:mac_zip_postprocess].call target_folder, target_file
+    else
+      puts 'Including licenses in mac .zip'
```

Comment:
I'd write `macOS` just to clarify

Reply:
I'm pretty sure it was like this before. Also I refer to mac as mac in plenty of other places, also the ruby gem refers to it like `OS.mac?`